### DISCUSSION TITLE
JobTrackr v1.1.0 Released

### DISCUSSION THREAD

**Discussion by mahimdashora** [SoftwareEngNoobs/jobtrackr#7] [Announcements]
🎉 JobTrackr v1.1.0 Released! 🚀
We’re thrilled to announce the official release of JobTrackr v1.1.0, the ultimate job application tracking tool! 🎯

What’s New in v1.1.0?
✅ Core Features:

Track your job applications effortlessly with intuitive workflows.
Organize job applications by status: Applied, Interviewing, Offer, or Rejected.
✨ New Features
1️⃣ Drag-and-Drop Kanban Boards
Organize your applications visually with our intuitive Kanban board.
Move job applications across stages such as Applied, Interviewing, Offer, or Rejected effortlessly.
Add, edit, and customize columns to fit your workflow.
Save time and track progress in a format that feels natural.
2️⃣ Cover Letter Support
Upload and store cover letters for each job application.
Create new cover letters using our rich-text editor, with templates to speed up your writing process.
Download and export cover letters directly from JobTrackr.

Built with a clean and modern interface for seamless user experience.
Stay tuned for updates and let us know how JobTrackr can be even better for you! 💡

Thank you to everyone who contributed to this release. Your feedback and support mean the world to us. ❤️

Happy job tracking!
– The JobTrackr Team

**Comment by VarunMK** [comment]
Looks Great!!!
